packages/xrpl/test/integration/submitAndWait.test.ts

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable @typescript-eslint/no-misused-promises -- supposed to return a promise here */
+/* eslint-disable no-restricted-syntax -- not sure why this rule is here, definitely not needed here */
```

Comment:
Should this be removed then?

Reply:
This should have never been added, someone was just not wanting to deal with it. I fixed the code.